README.md

```diff
@@ -4,14 +4,59 @@ Isso – *Ich schrei sonst* – is a lightweight commenting server written in
 Python and JavaScript. It aims to be a drop-in replacement for
 [Disqus](http://disqus.com).
 
-See **[posativ.org/isso](http://posativ.org/isso/)** for more details and
-documentation.
+## Features
 
-![Isso in Action](http://posativ.org/~tmp/isso-sample.png)
+- **Comments written in Markdown**  
+  Users can edit or delete own comments (within 15 minutes by default).
+  Comments in moderation queue are not publicly visible before activation.
+- **SQLite backend**  
+  *Because comments are not Big Data.*
+- **Disqus & WordPress Import**  
+  You can migrate your Disqus/WordPress comments without any hassle.
+- **Configurable JS client**  
+  Embed a single JS file, 65kB (20kB gzipped) and you are done.
 
-## License
-MIT, see [LICENSE](LICENSE).
+See **[posativ.org/isso](http://posativ.org/isso/)** for a **live demo**, more
+details and [documentation](https://posativ.org/isso/docs/).
+
+## Screenshot
+
+![Isso in Action](https://user-images.githubusercontent.com/10212877/167268553-3f30b448-25ff-4850-afef-df2f2e599c93.png)
+
+## Geting started
+
+**Requirements**
+- Python 3.6+ (+ devel headers)
```

Comment:
Why do we need devel headers or a C compiler? @ix5

Reply:
This was a copy-paste from the docs. Probably necessary for misaka and CFFI things?

Reply:
Devel headers were added way back in d1a0b3f6, I just happen to show up in the blame logs because I bumped versions a couple of times and moved files ;)

As for the validity, maybe we ought to collect empirical data by removing devel headers from our respective systems and seeing whether a full installation in a clean virtualenv (including misaka CFFI compilation) still works.